autoload/lightline/bufferline.vim

```diff
@@ -4,37 +4,38 @@
-let s:unicode_symbols     = get(g:, 'lightline#bufferline#unicode_symbols', 0)
+let s:dirsep                = fnamemodify(getcwd(),':p')[-1:]
+let s:filename_modifier     = get(g:, 'lightline#bufferline#filename_modifier', ':.')
+let s:min_buffer_count      = get(g:, 'lightline#bufferline#min_buffer_count', 0)
+let s:number_map            = get(g:, 'lightline#bufferline#number_map', {})
+let s:composed_number_map   = get(g:, 'lightline#bufferline#composed_number_map', {})
+let s:shorten_path          = get(g:, 'lightline#bufferline#shorten_path', 1)
+let s:smart_path            = get(g:, 'lightline#bufferline#smart_path', 1)
+let s:show_number           = get(g:, 'lightline#bufferline#show_number', 0)
+let s:number_separator      = get(g:, 'lightline#bufferline#number_separator', ' ')
+let s:ordinal_separator     = get(g:, 'lightline#bufferline#ordinal_separator', '')
+let s:unnamed               = get(g:, 'lightline#bufferline#unnamed', '*')
+let s:reverse_buffers       = get(g:, 'lightline#bufferline#reverse_buffers', 0)
+let s:reverse_buffer_icons  = get(g:, 'lightline#bufferline#reverse_buffer_icons', 0)
+let s:right_aligned         = get(g:, 'lightline#bufferline#right_aligned', 0)
+let s:enable_devicons       = get(g:, 'lightline#bufferline#enable_devicons', 0)
+let s:enable_nerdfont       = get(g:, 'lightline#bufferline#enable_nerdfont', 0)
+let s:unicode_symbols       = get(g:, 'lightline#bufferline#unicode_symbols', 0)
 if s:unicode_symbols == 0
-  let s:modified          = get(g:, 'lightline#bufferline#modified', ' +')
-  let s:read_only         = get(g:, 'lightline#bufferline#read_only', ' -')
-  let s:more_buffers      = get(g:, 'lightline#bufferline#more_buffers', '...')
+  let s:modified            = get(g:, 'lightline#bufferline#modified', ' +')
+  let s:read_only           = get(g:, 'lightline#bufferline#read_only', ' -')
+  let s:more_buffers        = get(g:, 'lightline#bufferline#more_buffers', '...')
 else
-  let s:modified          = get(g:, 'lightline#bufferline#modified', ' ✎')
-  let s:read_only         = get(g:, 'lightline#bufferline#read_only', ' ')
-  let s:more_buffers      = get(g:, 'lightline#bufferline#more_buffers', '…')
+  let s:modified            = get(g:, 'lightline#bufferline#modified', ' ✎')
+  let s:read_only           = get(g:, 'lightline#bufferline#read_only', ' ')
+  let s:more_buffers        = get(g:, 'lightline#bufferline#more_buffers', '…')
 endif
 if exists('g:lightline.component_raw.buffers')
-  let s:component_is_raw  = g:lightline.component_raw.buffers
+  let s:component_is_raw   = g:lightline.component_raw.buffers
 else
-  let s:component_is_raw  = 0
+  let s:component_is_raw   = 0
 endif
-let s:clickable           = has('tablineat') && s:component_is_raw ? get(g:, 'lightline#bufferline#clickable', 0) : 0
+let s:clickable            = has('tablineat') && s:component_is_raw ? get(g:, 'lightline#bufferline#clickable', 0) : 0
```

Comment:
Please revert this changes and instead add the following line below the definition of the `s:enable_nerdfont` variable:
```viml
let s:icon_position       = get(g:, 'lightline#bufferline#icon_position', 'left')
```